searchsploit

```diff
@@ -11,7 +11,7 @@
 
 
 ## OS settings
-gitpath="/usr/share/exploitdb"
```

Comment:
I agree. As the author hasn't responded, will open a new pull request afterwards to fix it (and bump the version number)